ros2_batch_job/__main__.py

```diff
@@ -218,10 +218,11 @@ def build_and_test(args, job):
     print('# BEGIN SUBSECTION: ament build')
     # Now run ament build
     ret_build = job.run([
-        '"%s"' % job.python, '-u', '"%s"' % ament_py, 'build', '--build-tests',
+        '"%s"' % job.python, '-u', '"%s"' % ament_py, 'build',
+        '"%s"' % args.sourcespace
```

Comment:
copy-paste has caused a syntax error here, at L245 and at L49 of packaging.py :smile: